### PR TITLE
Diaspora: Set database server to postgres

### PR DESCRIPTION
**Pull request by JosephKiranBabu**

### CODE REVIEW

File: actions/diaspora [hunk 1]
```diff
@@ -20,11 +20,14 @@
 Configuration helper for diaspora* pod.
 """
 
+import os
 import argparse
 import subprocess
 
 from plinth import action_utils
 
+GEMFILE = '/usr/share/diaspora/Gemfile'
```
Comment (jvalleroy) [inline actions/diaspora]: I don't think we should modify any files in /usr, except during installation.

Why is this change needed? Is it a bug in the diaspora package?

**Comment by JosephKiranBabu** [comment]
This is not the right way to set the database to postgresql.
Any bug fix for setting the database should go into the deb package and not into Plinth. 
Closing this.
